cvat-ui/src/components/annotation-page/top-bar/annotation-menu.tsx

```diff
@@ -137,15 +142,15 @@ function AnnotationMenuComponent(): JSX.Element {
                                             min={0}
                                             max={stopFrame}
                                             onChange={(value) => {
-                                                removeFrom = value;
+                                                removeFrom = value ?? undefined;
```

Comment:
Why do we need these two changes? 
If value is not specified, it is already `undefined` or `null`.